docs/pages/ktable.vue

```diff
@@ -23,7 +23,18 @@
       title="Usage"
       anchor="#usage"
     >
-      <!--Non-Sortable Table-->
+      <h3>Table caption</h3>
+
+      <p>
+        A caption is required via the <code>caption</code> prop. Provide a concise and translated
+        string that accurately describes the table’s content. See
+        <a
+          href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption#table_with_caption"
+          target="_blank"
+        >MDN</a>
+        for more.
```

Comment:
I was earlier thinking about adding reference to an example from Kolibri here that makes use of translated strings for caption, but that brought me to the observation, that as of now, none of the tables (Facility/Users) make use of translated strings for captions. Now, I'm doubtful, but is the use of translated strings even needed here, or can the corresponding screen readers handle them on their own?

Reply:
Ah yes we could reference Kolibri from here, that's a good idea. Or perhaps even better, our [Internationalization guide](https://kolibri-dev.readthedocs.io/en/develop/i18n.html#internationalization)? That link would be more stable than linking codebase, and it has examples + additional explanation.

I don't think screenreaders will translate captions on their own, and we need to use translated strings as for anything else. Nevertheless, I double-checked and Kolibri `KTables` have translated caption, such as [here](https://github.com/learningequality/kolibri/blob/cb022ad0dd6295365df48da2402643ac718db349/kolibri/plugins/facility/assets/src/views/UserPage/index.vue#L66). If I am missing some places, would you reference? We would need to fix that in Kolibri.

Reply:
Ahh, my bad. When I did the KTable transition initially, I hadn't done this translation thing. Maybe, it was added in some later PRs. Apologies for the confusion caused :|
Also, adding the internationalization guide sounds like a great idea!

Reply:
No problem